src/zarr/buffer/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""
+Public API for implementations of the Zarr Buffer interface.
```

Comment:
```suggestion
Implementations of the Zarr Buffer interface.
```
If it's in the docs, it's implicitly public API already